.env.example

```
@@ -0,0 +1 @@
+BASE_URL = https://dogapi.dog/api/v2
```

```suggestion
BASE_URL=https://dogapi.dog/api/v2
```

Unix env vars, [point 2.2](https://www3.ntu.edu.sg/home/ehchua/programming/howto/Environment_Variables.html).